.github/workflows/nightly-e2e-tests-matrix.yml

```
@@ -148,3 +148,36 @@ jobs:
+  test_execution_playwright:
+    # Name must start with "test execution / " -- the setup shards' own wait
+    # step (runner-e2e-tests-codeceptjs-remote-nightly-setup.yml) polls jobs
+    # by that literal prefix to count consumers; anything else is invisible
+    # to it and the shard can finish/terminate while this job still needs it.
+    name: "test execution / playwright ${{ matrix.tags_for_tests }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - tags_for_tests: '@annotations'
+    uses: ./.github/workflows/runner-e2e-tests-playwright-remote-nightly-tests.yml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/runner-e2e-tests-playwright-remote-nightly-tests.yml": open /home/jailuser/git/.github/workflows/runner-e2e-tests-playwright-remote-nightly-tests.yml: no such file or directory

(workflow-call)